src/coordinator/tsdb/remote/codecs_test.go

```diff
@@ -39,16 +39,17 @@ import (
 var (
 	name0    = "regex"
 	val0     = "[a-z]"
-	spec0    = "specs"
-	valList0 = []float32{1.0, 2.0, 3.0}
-	mps0     = int32(100)
-	time0    = "2000-02-06T11:54:48+07:00"
+	valList0 = &rpc.Datapoints{
+		Datapoints:      []*rpc.Datapoint{{1, 1.0}, {2, 2.0}, {3, 3.0}},
+		FixedResolution: false,}
```

Comment:
Could we get a small conversion test with `FixedResolution: true`?

Reply:
helped me catch a bug :)